### ISSUE TITLE
Could you share the training code and the weights used in the final test code?

### ISSUE THREAD

**Issue by 1hzf** [CVhfut/PCOD_1200#6]
We reproduced train.py and the related loss settings using AI based on the descriptions in the paper, but the best result obtained on 230 test images is slightly different from that reported in the paper

<img width="275" height="108" alt="Image" src="https://github.com/user-attachments/assets/2e387d9a-8a56-4179-9c97-e2b819a19bbf" />

[train.py](https://github.com/user-attachments/files/27154176/train.py)
